### PR TITLE
adding persona.md doc for discussion

### PR DESCRIPTION
**Pull request by ash-sxn**
Hi,
@jmMeessen
@gounthar 
@berviantoleo 
What do you think about this? please suggest your thoughts, any changes or suggestions.

### CODE REVIEW

File: PERSONA.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+For this person the the fully automated docker compose files and scripts are needed like you created in PR which works by only one command or maybe git clone(like in kubernetes part [here](https://www.jenkins.io/doc/book/installing/kubernetes/#jenkins-kubernetes-manifest-files)) from some repo before running docker compose up 
+
+## Second Persona 
+This person has a understanding of Docker and is interested in using Docker Compose to customise their Jenkins setup. They may want to learn more about how Docker and Jenkins work together and are willing to spend time learning about the technical details. They may be interested in optimising their workflow and making the most of the tools available. They may want to have more control over their setup and are willing to spend time customising it to meet their needs. and they want to know what exactly is running on their system before pressing enter
+This is the person whom the simple docker compose files are aimed towards, to make it easy to understand how docker compose works with Jenkins
```
Comment (gounthar) [inline PERSONA.md:9]:
```suggestion
# Persona

## First Persona

This persona may have little or no knowledge about Docker Compose files or how they function.
They may not be interested in delving into the technical intricacies and simply want to use Jenkins without worrying about the setup process.
Although they may not have much familiarity with Jenkins at the moment, they are eager to explore its functionality, appearance, and user experience.
Their goal is to quickly and effortlessly start using Jenkins without investing significant time in setup and configuration.
To cater to this persona, fully automated `docker-compose` files and scripts are required, allowing them to set up Jenkins with a single command or by cloning a repository (similar to the Kubernetes part mentioned [here](https://www.jenkins.io/doc/book/installing/kubernetes/#jenkins-kubernetes-manifest-files)) before running `docker compose up`.

## Second Persona

This persona possesses a solid understanding of Docker and is keen on utilizing Docker Compose to customize their Jenkins setup.
They want to deepen their knowledge of how Docker and Jenkins integrate and are willing to invest time in understanding the technical aspects.
Their objective is to optimize their workflow and leverage the available tools to their fullest potential.
They desire greater control over their setup and are prepared to spend time tailoring it to align with their requirements.
This persona wants to have a clear understanding of the components running on their system before executing any commands.
The simple `docker-compose` files cater to their needs by providing transparency and facilitating their comprehension of how `docker compose` interacts with Jenkins.
```

File: PERSONA.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Persona
+
```
Comment (berviantoleo) [inline PERSONA.md:2]: Could we have a table to summarize the personas?

Example:

| Persona | Docker | Docker Compose | Linux |
|:--------:|:---------:|:--------------------:|:-----:|
| First | No or Little Knowledge | No or Little Knowledge | No | 
| Second | Have Knowledge | Have Knowledge | Have |

It doesn't need to map all tools or concepts.

**Comment by jmMeessen** [comment]
I fear that the personae description will not help us list the features that need to be implemented. I believe that the introduction of the personae description should describe what he wants to  achieve. In the same way as in a UML use case.  You have the material later in the paragraph, but unstructured. Try to start in a one liner what he wants/needs.

A tip to know how to describe this efficiently: you are entering an elevator together with the person who is funding your project. You only a handful seconds to sell your idea. It must be clear, crisp, punchy.

The description of the characteristics of the personae are good although you are insisting too much on docker knowledge.

**Comment by gounthar** [comment]
@ash-sxn please try to address the changes requested by @jmMeessen . 👍

**Review by berviantoleo — APPROVED** [review]

**Comment by ash-sxn** [comment]
Hi,
@gounthar @jmMeessen 
Please take a look, does this work? 
Should I add anything else to the table or any other point?

**Review by jmMeessen — APPROVED** [review]
Much better. Thank you for these improvements.

**Pull request merged**